k8s/kube-prometheus/conbench-flavor.jsonnet

```diff
@@ -0,0 +1,51 @@
+// Note(JP): Starting point was a raw copy of
+// https://github.com/prometheus-operator/kube-prometheus/blob/v0.12.0/example.jsonnet
+// I then tried to understand some of
+// https://github.com/prometheus-operator/kube-prometheus/tree/main/docs/customizations
+// to do some modifications (mainly: inject custom dashboard)
+// I use use auto-format in VSCode with the 'jsonnet Formatter' extension
+local kp =
+  (import 'kube-prometheus/main.libsonnet') +
```

Comment:
If you have not heard of jsonnet until today:
- https://jsonnet.org/
- https://jsonnet.org/learning/tutorial.html
- https://github.com/google/jsonnet